README

```diff
@@ -40,8 +40,9 @@ GNU versions of these tools are recommended.
 
 Installation
 ------------
-Installing Geany is done by the following three commands:
-$ ./configure
+Installing Geany is done by the following commands:
+$ libtoolize
+$ ./autogen.sh
```

Comment:
And `autogen.sh` is only needed first time after a new git clone, for tarballs or the second time only `./configure` is needed.